src/app/app-routing.module.ts

```diff
@@ -11,6 +12,7 @@ const routes: Routes = [
   //{path:'',component: AppComponent,pathMatch: 'full'},
   {path: '' , component: IndexComponent},
   {path:':id/itineraries', component: ItinerarylistComponent,pathMatch: 'full'},
+  {path:':id/itinerary', component: ItineraryViewContoller,pathMatch: 'full'},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181090)
```suggestion
  {path:":id/itinerary", component: ItineraryViewContoller,pathMatch: 'full'},
```